frontend/src/components/viewers/VisualizationCreator.test.tsx

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { PlotType } from './Viewer';
+import VisualizationCreator, { VisualizationCreatorConfig } from './VisualizationCreator';
+import { ApiVisualizationType } from '../../apis/visualization';
+
+describe('VisualizationCreator', () => {
+  it('does not break when no configs are provided', () => {
```

Comment:
what _does_ it do though, does it display an empty space? is it visible? if there's no better description, I'd still say something like it `handles empty configs` or something